setup.py

```diff
@@ -28,7 +28,7 @@ def package_files(directory):
     "../mlflow/store/db_migrations/alembic.ini",
     "../mlflow/temporary_db_migrations_for_pre_1_users/alembic.ini",
 ]
-extra_files = ["../mlflow/ml-package-versions.yml"]
+extra_files = ["../mlflow/ml-package-versions.yml", "../mlflow/pyspark/ml/log_model_allowlist.txt"]
```

Comment:
Without this, the mlflow package won't include the log_model_allowlist.txt file.

Reply:
```suggestion
extra_files = ["ml-package-versions.yml", "pyspark/ml/log_model_allowlist.txt"]
```

Does this work?